pages/blog/async_standards_compare.md

```diff
@@ -0,0 +1,97 @@
+---
+type: Engineering
+cover: /img/posts/async_standards_compare/devops-specs_pic-01.webp
+title: "AsyncAPI, CloudEvents, OpenTelemetry: Which Event-Driven Specs Should Your DevOps Include?"
+date: 2021-05-21T06:00:00+01:00
+featured: true
+tags:
+   - asyncapi
+   - opentelemetry
+   - EDA
+   - CloudEvents
+authors:
+   - name: Jesse Menning
+     photo: /img/avatars/jmenning.webp
+     link: https://twitter.com/jmenning
```

Comment:
I see you are not that active on twitter, feel free to change to linkedin or something else if you want, it is totally up to you where this link takes people. Just letting you know as there is still time to change it before friday

Reply:
changed to linked in